### PR TITLE
Set up GraphQL, Route & add Schema

### PR DESCRIPTION
**Pull request by Asmaathabet**
- set up Graphql 
- add The Route 
- add Schema

### CODE REVIEW

File: server/schema/index.js [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+    city: { type: new GraphQLNonNull(GraphQLString) },
+    postCode: { type: new GraphQLNonNull(GraphQLString) },
+    type: { type: new GraphQLNonNull(GraphQLString) },
+    priceRange: { type: new GraphQLNonNull(GraphQLString) },
+    language: { type: new GraphQLNonNull(new GraphQLList(GraphQLString)) },
+    insurance: { type: new GraphQLNonNull(new GraphQLList(GraphQLString)) },
+    approch: { type: new GraphQLNonNull(GraphQLString) },
+    avalibility: { type: new GraphQLNonNull(GraphQLString) },
+    image: { type: new GraphQLNonNull(GraphQLString) },
+    remote: { type: GraphQLBoolean },
```
Comment (AbdallahAmmar96) [inline server/schema/index.js]: remote is also not null

File: server/schema/index.js [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+    skype: { type: GraphQLString },
+  }),
+});
+
+const RootQuery = new GraphQLObjectType({
+  name: 'therapistQuery',
+  fields: {
+    therapists: {
+      type: therapistType,
+    //   resolve(parent, args) {
```
Comment (AbdallahAmmar96) [inline server/schema/index.js]: commented code

File: server/schema/index.js [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+        city: { type: new GraphQLNonNull(GraphQLString) },
+        postCode: { type: new GraphQLNonNull(GraphQLString) },
+        type: { type: new GraphQLNonNull(GraphQLString) },
+        priceRange: { type: new GraphQLNonNull(GraphQLString) },
+        language: { type: new GraphQLNonNull(new GraphQLList(GraphQLString)) },
+        insurance: { type: new GraphQLNonNull(new GraphQLList(GraphQLString)) },
+        approch: { type: new GraphQLNonNull(GraphQLString) },
+        avalibility: { type: new GraphQLNonNull(GraphQLString) },
+        image: { type: new GraphQLNonNull(GraphQLString) },
+        remote: { type: GraphQLBoolean },
```
Comment (AbdallahAmmar96) [inline server/schema/index.js]: not null

File: server/app.js [hunk 4]
```diff
@@ -17,7 +18,10 @@ app.use(formData.parse());
 app.use(express.urlencoded({ extended: false }));
 
 app.use(express.static(join(__dirname, '..', 'client', 'build')));
-app.use('/api/v1', router);
+app.use('/graphql', graphqlHTTP({
```
Comment (AbdallahAmmar96) [inline server/app.js]:
```suggestion
app.use('/shrinc-graphql', graphqlHTTP({
```
```suggestion
app.use('/shrinc', graphqlHTTP({
```

File: server/schema/index.js [hunk 5]
```diff
@@ -0,0 +1,79 @@
...
+  GraphQLObjectType,
+  GraphQLString,
+  GraphQLSchema,
+  GraphQLID,
+  GraphQLList,
+  GraphQLBoolean,
+  GraphQLNonNull,
+} = graphql;
+
+const therapistType = new GraphQLObjectType({
```
Comment (AbdallahAmmar96) [inline server/schema/index.js]: put the types in another file and import it here

Comment (Asmaathabet) [reply server/schema/index.js]: do you mean fields ?

File: server/schema/index.js (hunk 2 above)
Comment (Asmaathabet) [reply server/schema/index.js]: I wait for accepting Monogodb pull request to use the models here . I just commented it to allow pushing it without eslint errors.

**Pull request merged**